### ISSUE TITLE
Prediction on original data set

### ISSUE THREAD

**Issue by moabe84** [deepchem/deepchem#3193]
Hi Everyone,
I have a question. I'd like to do prediction on my original test data set. I've realized that I need first to transform back the data. I've tried many things, but so far I've not been able to find a way to do that. Any help or suggestions are greatly appreciated. Many thanks.

Here is my script:

featurizer = dc.feat.ConvMolFeaturizer()
features = featurizer.featurize(data['smiles'])

dataset = dc.data.NumpyDataset(X = features, y = data['solubility'])

splitter = dc.splits.RandomSplitter()
train_dataset, valid_dataset, test_dataset = splitter.train_valid_test_split(dataset)

transformer = dc.trans.NormalizationTransformer(transform_y = True, dataset = dataset)
train_dataset = transformer.transform(train_dataset)
valid_dataset = transformer.transform(valid_dataset)
test_dataset = transformer.transform(test_dataset)

metric = dc.metrics.Metric(dc.metrics.pearson_r2_score, np.mean)
model = dc.models.GraphConvModel(n_tasks = 1, batch_size = 64, learning_rate = 0.001, mode = 'regression')
model.fit(train_dataset, nb_epoch=20)

test_preds = model.predict(test_dataset)

**Comment by moabe84** [comment]
It's resolved.
